packages/functions/test/wait-until.test.ts

```diff
@@ -26,15 +24,23 @@ test.each([null, undefined, {}])(
   'waitUntil does not throw an error when context is %s',
   input => {
     const promise = Promise.resolve();
-    globalThis[Symbol.for('@vercel/request-context')] = input;
+    globalThis[
+      // @ts-ignore
```

Comment:
consider using
```
// @ts-expect-error
```

Reply:
or declare a type

```typescript
const symbol = Symbol.for('@vercel/request-context');
const g: typeof globalThis & { [symbol]?: unknown } = globalThis;
```

and then you can mutate `g` instead of `globalThis`